src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int TOTAL_FIGURE = 5;
+    public static final String DEFAULT_COLOR = "WHITE";
+    public static final int DEFAULT_RADIUS = 10;
+    public static final int LIMIT = 100;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int indexOfFigure = random.nextInt(TOTAL_FIGURE);
+        switch (indexOfFigure) {
+            case 0: return new Square(colorSupplier.getRandomColor(), random.nextInt(LIMIT));
+            case 1: return new Rectangle(colorSupplier.getRandomColor(),
+                    random.nextInt(LIMIT),
+                    random.nextInt(LIMIT));
+            case 2: return new RightTriangle(colorSupplier.getRandomColor(),
+                    random.nextInt(LIMIT),
+                    random.nextInt(LIMIT));
+            case 3: return new Circle(colorSupplier.getRandomColor(),
+                    random.nextInt(LIMIT));
+            case 4: return new IsoscelesTrapezoid(colorSupplier.getRandomColor(),
+                    random.nextInt(LIMIT),

```

Comment:
consider going through the enum of shapes for example to make It more clear what cases do you have
for example 
`case CIRCLE: , not case 3: `

Reply:
it's an optional one, I can accept this and discuss it on Q&A